tasks/transform_conformance/ts_fixutures/enums__awaitAndYield.ts

```diff
@@ -0,0 +1,9 @@
+async function* test(x) {
+	var E;
+	(E => {
+		const foo = await x;
```

Comment:
tsc will report error with this case.
[playground](https://www.typescriptlang.org/play?target=6#code/PTAEAEBcEMCcHMCmkBcoCiBlATABgIwCcAUCBAHYD2AKgJ4AOiAzgILkAmmtAtgEaUAbJmkiwAromLQmtcgGNQAMzHzIAS0rkAVKEjNIACgAeaAAqxK3NU0QAecmL6JYAPgCUoAN7FQv0IgduDC8fPzDFSkpQAF5QaAB3aDVIUCMAGlCw315oAC8Y0Fo1RAF2UHwMsIBfYiqgA)